src/gui/torrentcontentmodelfolder.cpp

```diff
@@ -147,10 +147,19 @@ void TorrentContentModelFolder::recalculateProgress()
         tRemaining += child->remaining();
     }
 
-    if (!isRootItem() && (tSize > 0))
+    if (!isRootItem())
```

Comment:
I believe it is enough to initialize m_progress as 1.0 in TorrentContentModelFolder constructor.